config/environments/development.rb

```diff
@@ -25,10 +25,11 @@
     
   config.action_mailer.delivery_method = :smtp
   config.action_mailer.smtp_settings = {
-    address:              'mail.metamaps.cc',
-    port:                 587,
-    user_name:            'team@metamaps.cc',
-    password:             'RcxX+s:fht49UX',
+    address:       ENV['SMTP_SERVER'],
+    port:          ENV['SMTP_PORT'],
+    user_name:     ENV['SMTP_USERNAME'],
+    password:      ENV['SMTP_PASSWORD'],
+    #domain:        ENV['SMTP_DOMAIN']
```

Comment:
My understanding is its a setting to declare the where the mail is coming
from, to increase deliverability / minimize bounce.
May only be relevant if you have a separate mail server

http://guides.rubyonrails.org/action_mailer_basics.html#action-mailer-configuration

Let me know if you want it out.
On 5/03/2015 3:14 am, "Connor Turland" <notifications@github.com> wrote:

> In config/environments/development.rb
> <https://github.com/metamaps/metamaps_gen002/pull/406#discussion_r25776623>
> :
>
> > @@ -25,10 +25,11 @@
> >
> >    config.action_mailer.delivery_method = :smtp
> >    config.action_mailer.smtp_settings = {
> > -    address:              'mail.metamaps.cc',
> > -    port:                 587,
> > -    user_name:            'team@metamaps.cc',
> > -    password:             'RcxX+s:fht49UX',
> > +    address:       ENV['SMTP_SERVER'],
> > +    port:          ENV['SMTP_PORT'],
> > +    user_name:     ENV['SMTP_USERNAME'],
> > +    password:      ENV['SMTP_PASSWORD'],
> > +    #domain:        ENV['SMTP_DOMAIN']
>
> Looks good, but what's this line for?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/metamaps/metamaps_gen002/pull/406/files#r25776623>.
>